modules/on-prem/pages/activate.adoc

```diff
@@ -1,4 +1,4 @@
-= Activate
+= Activate ML Workbench
 
 After the on-prem ML Workbench setup is complete, the next step is to activate ML Workbench on your on-prem TigerGraph database instance.
 This is a prerequisite for some ML Workbench core functionality.
```

Comment:
core -> enterprise?

Reply:
oh, current activator is for both developer and enterprise version. never mind, this is good for now.